lib/active_merchant/billing/gateways/mercury.rb

```diff
@@ -90,7 +90,7 @@ def build_non_authorized_request(action, money, credit_card, options)
           xml.tag! "Transaction" do
             xml.tag! 'TranType', 'Credit'
             xml.tag! 'TranCode', action
-            if action == 'PreAuth' || action == 'Sale'
+            if (options[:allow_partial_auth] == true) && (action == 'PreAuth' || action == 'Sale')
```

Comment:
I agree.  Not sure what I was thinking.  

I had thougts about people passing in "false", but they shouldn't do that.  All in all, not a very vulcan moment.  Truthiness is better and simpler.  Will fix.